app/src/main/java/com/example/trempelapp/presentationLayer/logInScreen/LogInActivity.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.trempelapp.presentationLayer.logInScreen
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.example.trempelapp.R
+
+class LogInActivity : AppCompatActivity() {
```

Comment:
Create both BaseActivity and BaseFragment and extend it

Reply:
done